google-api-go-generator/gen_test.go

```diff
@@ -2,6 +2,9 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
+//go:build go1.19
```

Comment:
Why is this needed?

Reply:
The goldens for google-api-go-generator were failing for 1.19, so I updated them, and now the new goldens fail for the earliest version (and any other version < 1.19).

Reply:
Ah sure, since they include godoc. Interesting, would have never thought of that. Thanks!